helpers/kubernetes/cluster.go

```diff
@@ -393,6 +425,40 @@ func (c *Cluster) ListPods(ctx context.Context, namespace, selector string) (*v1
 	return podList, nil
 }
 
+// ListJobs returns the list of currently scheduled or running Jobs in `namespace` with the given selector
+func (c *Cluster) ListJobs(ctx context.Context, namespace, selector string) (*apibatchv1.JobList, error) {
+	listOptions := metav1.ListOptions{}
+	if len(selector) > 0 {
+		listOptions.LabelSelector = selector
+	}
+	jobList, err := c.Kubectl.BatchV1().Jobs(namespace).List(ctx, listOptions)
```

Comment:
unrelated, but it's strange to call the clientset 'kubectl'.

Reply:
Inherited from way long in the past. Will not fix. Please create a ticket if the itch about it strong enough.